qiskit_nature/problems/second_quantization/problem_second_quant_ops.py

```diff
@@ -0,0 +1,53 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""Problem Second Quantized Operators"""
+from typing import List
+
+from qiskit_nature.operators.second_quantization import SecondQuantizedOp
+
+
+class ProblemSecondQuantOps():
```

Comment:
I think it would be nice to stay consistent with the other appearances of "quantized", i.e. since we have `SecondQuantizedOp` rather have `ProblemSecondQuantizedOps`?